proto/cosmos/authz/v1beta1/authz.proto

```diff
@@ -35,3 +35,26 @@ message AuthorizationGrant {
   google.protobuf.Any       authorization = 1 [(cosmos_proto.accepts_interface) = "Authorization"];
   google.protobuf.Timestamp expiration    = 2 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
 }
+
+// DelegateAuthorization allows the grantee to delegate tokens to provided set of validators
+// on behalf of the granter's account.
+message DelegateAuthorization {
+  option (cosmos_proto.implements_interface) = "Authorization";
+
+  // validator_address specifies list of validator addresses to whom grantee can delegate tokens on behalf of granter's account.
+  repeated string validator_address = 1;
```

Comment:
Should we take an "allow list" and "deny list" type approach? Just a thought, but this is also okay.

Reply:
That's a good idea. Let's have 2 lists